src/server/v3/resources/jobs.py

```diff
@@ -627,14 +627,14 @@ def post(self):
 
         current_app.logger.info(f"ARTIFACT_RECORD: {artifact_record}")
 
-        # both images and recipes have a platform specified - shift into the job
-        new_job.platform = artifact_record.platform
-        current_app.logger.info(f"PLATFORM: {new_job.platform}")
+        # both images and recipes have a architecture specified - shift into the job
```

Comment:
Nit: a --> an

Reply:
Fixed. Also fixed the same thing in v2/resources/jobs.py.